src/js/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
```

Comment:
now we only ever have a single import of `axios`, we can change out at anytime for another package if needed